BUILD.md

```diff
@@ -353,21 +353,21 @@ To build for Intel GPU, replace dnnl_opencl_root with the path of the Intel SDK
 See more information on the OpenVINO Execution Provider [here](./docs/execution_providers/OpenVINO-ExecutionProvider.md).
 
 #### Prerequisites
-1. Install the Intel<sup>®</sup> Distribution of OpenVINO<sup>TM</sup> Toolkit **Release 2021.1** for the appropriate OS and target hardware :
+1. Install the Intel<sup>®</sup> Distribution of OpenVINO<sup>TM</sup> Toolkit **Release 2021.2** for the appropriate OS and target hardware :
```

Comment:
General comment/question.
What is the back compat story for ORT + OpenVINO? How many versions back are we supporting? 
(currently it is 5, going back to 2020.2)
I suggest we cap it at a reasonable number. i.e. N-2  , otherwise it feels like a lot of maintenance overhead.

Reply:
I agree, it is the conversation we need to have and decide on.